VGG16_Net.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>nice use of epsilon!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/Skin-Cancer-Classification/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>